host_vars/dalite.harvardx.harvard.edu/public.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+MYSQL_DALITE_HOST: "149.202.183.147"
```

Comment:
This setting should be moved to the private vars? The (public) default should be non-functional imho, to avoid people reusing this from hammering an existing server if they don't touch the config.